src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticAnalysisContext.cs

```diff
@@ -1151,10 +1151,15 @@ public struct OperationAnalysisContext
         /// </summary>
         public ISymbol ContainingSymbol => _containingSymbol;
 
+        /// <summary>
+        /// <see cref="CodeAnalysis.SemanticModel"/> that can provide semantic information about the <see cref="IOperation"/>.
```

Comment:
>that can provide semantic information about the <see cref="IOperation"/> [](start = 53, length = 72)

This sounds strange. SemanticModel doesn't provide information about IOperation nodes. Is this SemanticModel used to obtain the IOperation node? If that is the case, consider saying that instead.

Reply:
Not sure what is good wording here, grabbed the doc comment from `SyntaxNodeAnalysisContext`

Reply:
Is `<see cref="CodeAnalysis.SemanticModel"/> for the <see cref="Document" /> that contains the <see cref="IOperation"/>.` a good wording?

Reply:
> Is ```<see cref="CodeAnalysis.SemanticModel"/> for the <see cref="Document" /> that contains the <see cref="IOperation"/>. a good wording?```

If we go with implementation ```(operation as Operation)?.SemanticModel```, then I would say "that created" rather than "that contains".

---
In reply to: [182549515](https://github.com/dotnet/roslyn/pull/26187#discussion_r182549515) [](ancestors = 182549515)